OpenRA.Mods.Common/Projectiles/GravityBomb.cs

```diff
@@ -36,6 +36,8 @@ public class GravityBombInfo : IProjectileInfo
 
 		public readonly WDist Speed = WDist.Zero;
 
+		public readonly WDist ForwardSpeed = WDist.Zero;
```

Comment:
I'd like to see this be combined with `Speed` into a single `WVec Velocity` property.  A pair of weapons with opposite Y velocities could then be used to bomb out both sides of an aircraft.